api/server/volume.go

```diff
@@ -598,5 +697,9 @@ func (vd *volAPI) Routes() []*Route {
 		{verb: "POST", path: snapPath("", volume.APIVersion), fn: vd.snap},
 		{verb: "GET", path: snapPath("", volume.APIVersion), fn: vd.snapEnumerate},
 		{verb: "POST", path: snapPath("/restore/{id}", volume.APIVersion), fn: vd.restore},
+		{verb: "GET", path: credsPath("/credslist", volume.APIVersion), fn: vd.credsList},
```

Comment:
credsPath("/credslist") can be simply credsPath("/endpoint").
Also - get without parameter is enumerate all
get with parameter is single get

Reply:
That was the the way i started. However unit tests with httptest does not work with that mode.

Reply:
@lpabon can you help @talakad  with UT questions ? thanks !

Reply:
@talakad I'm not sure what you mean. Let's talk tomorrow and see what we can do.